Makefile

```diff
@@ -39,6 +39,16 @@ download:
 	@echo Download go.mod dependencies
 	@go mod download
 
+## install-tools: Installing tools from tools.go
+install-tools: download
+	@echo Installing tools from tools.go
+	@cat tools.go | grep _ | awk -F'"' '{print $$2}' | xargs -tI % go install %
```

Comment:
`go install` will install the binary on user's $GOPATH/bin. If there's that same tool already installed I guess it'll overwrite any existing one (that might be another version by another project or installed by the user itself).

Do you think it might be a good idea to change the output path? (I guess for that go build has to be used instead of go install)

Reply:
I've seen go-bindata seems to be a used tool. Maybe placing it outside of $GOPATH/bin would make go generate not work appropriately though

Reply:
This is regular way of working with go tools. 

If there are issues with versioning, as you highlight, customizing the GOPATH env var is the way to go

Reply:
:+1: